db/types/operations/cast.py

```diff
@@ -597,6 +607,58 @@ def _get_timestamp_without_timezone_type_body_map():
 
 
 def _get_money_type_body_map():
+    """
+    Get SQL strings that create various functions for casting different
+    types to money.
+    We allow casting any number type to money, assuming currency is USD.
+    We allow casting any textual type to money, assuming currency is USD
```

Comment:
Is this accurate?  Isn't the function assuming the currrency symbol associated with the DB locale?  Keep in mind users can attach Mathesar to a preexisting database installation.